ott/tools/sinkhorn_divergence.py

```diff
@@ -173,6 +174,7 @@ def segment_sinkhorn_divergence(
     x: Array of input points, of shape [num_x, feature]. Multiple segments are
       held in this single array.
     y: Array of target points, of shape [num_y, feature].
+    cost_fn:
```

Comment:
Since you haven't added it to the arguments, this should not be here. That being said, I see why you wanted to add it. At the moment `cost_fn` is hidden in `kwargs` and passed on to the `geometry`. 

We could remove it from kwargs, and make it visible in the list of arguments, defaulting to None, and pass it to the point clouds when computing the `sinkhorn_divergence`. This way, if not None, we can compute the padding vector.

Reply:
the cost_fn in the doctoring is a typo form when I was working on a different solution

apologies for that - will remove it :)